plugins/gatsby-source-bxjs/episodesToDocuments.js

```diff
@@ -19,12 +20,12 @@ exports.markdownToDocuments = text => {
     .map(section => section.replace(/\r/g, ''))
     .filter(
       section =>
-        section && section.length > 0 && section.replace(/\n/g, '').length > 0
+        section && section.length > 0 && section.replace(/\n/g, '').length > 0,
     )
-    .flatMap(text => {
-      const [name, linksText] = text.split(/:\n/g);
+    .flatMap(text2 => {
```

Comment:
I get that it shadows the var in higher scope, but `text2` is a terrible var name 😅

Reply:
Actually airbnb lint rules showed that error, but standard ruleset are happy. Should I revert it back?